### PR TITLE
Remove redirects from /rhel to /insights based on doc for stage-beta

### PR DESCRIPTION
**Pull request by astrozzc**
JIRA: https://issues.redhat.com/browse/RHCLOUD-13060

Doc is here, RHEL Nav section:
https://docs.google.com/document/d/1NkAwv1JT22-iG-erPLMToJ3WGxadjVV8SVg-2j8WxCI/edit#heading=h.24lar9o0onc

### CODE REVIEW

**Comment by astrozzc** [comment]
Waiting for Nicole's response to add new redirects

**Comment by kruai** [comment]
LGTM, but please @ me when it's no longer WIP and I'll give it my stamp of approval

**Comment by astrozzc** [comment]
@kruai Thank you! Just added some new redirects that have to be done before summit. 
There is one place that needs attention: there are two similar redirects based on the doc: `openshift/subscriptions/openshift-container*` and `openshift/subscriptions/openshift-container/*`, not sure if the redirect would work. Could you help me to confirm based on your experience? Meanwhile, I will ask Nicole if the url are correct for these two.

**Comment by kruai** [comment]
@astrozzc Nope, you can't use a regex in the destination (it's only for matching). What scenarios are you planning for with this change?

**Comment by astrozzc** [comment]
> @astrozzc Nope, you can't use a regex in the destination (it's only for matching). What scenarios are you planning for with this change?

Thanks Asa! Updated

File: akamai/data/stage/base_rules.json [hunk 1]
```diff
@@ -360,7 +360,7 @@
                                         "options": {
                                             "matchOperator": "MATCHES_ONE_OF",
                                             "values": [
-                                                "/beta/rhel"
+                                                "/beta/openshift"
```
Comment (kruai) [inline akamai/data/stage/base_rules.json]: This redirect will only work for the exact path `/beta/openshift`. Is that intentional?

File: akamai/data/stage/base_rules.json [hunk 2]
```diff
@@ -424,7 +424,7 @@
                                         "options": {
                                             "matchOperator": "MATCHES_ONE_OF",
                                             "values": [
-                                                "/beta/rhel/vulnerability*"
+                                                "/beta/openshift/subscriptions"
```
Comment (kruai) [inline akamai/data/stage/base_rules.json]: This redirect will only work for the exact path `/beta/openshift/subscriptions`. Is that intentional?

File: akamai/data/stage/base_rules.json [hunk 3]
```diff
@@ -616,7 +616,391 @@
...
+                                        }
+                                    }
+                                ],
+                                "criteria": [
+                                    {
+                                        "name": "path",
+                                        "options": {
+                                            "matchOperator": "MATCHES_ONE_OF",
+                                            "values": [
+                                                "/beta/insights/ros"
```
Comment (kruai) [inline akamai/data/stage/base_rules.json]: This redirect will only work for the exact path `/beta/insights/ros`. Is that intentional?

File: akamai/data/stage/base_rules.json [hunk 4]
```diff
@@ -616,7 +616,391 @@
...
+                                        }
+                                    }
+                                ],
+                                "criteria": [
+                                    {
+                                        "name": "path",
+                                        "options": {
+                                            "matchOperator": "MATCHES_ONE_OF",
+                                            "values": [
+                                                "/beta/subscriptions/rhel-sw/all/*"
```
Comment (kruai) [inline akamai/data/stage/base_rules.json]: This match appears to duplicate line 459

**Review by kruai — CHANGES_REQUESTED** [review]
I just have a few things I want to confirm.

File: akamai/data/stage/base_rules.json (hunk 4 above)
Comment (astrozzc) [reply akamai/data/stage/base_rules.json]: Good catch!

File: akamai/data/stage/base_rules.json (hunk 1 above)
Comment (astrozzc) [reply akamai/data/stage/base_rules.json]: I guess so, in the [doc](https://docs.google.com/document/d/1NkAwv1JT22-iG-erPLMToJ3WGxadjVV8SVg-2j8WxCI) it says: 
Current: /openshift
New: /openshift/clusters

File: akamai/data/stage/base_rules.json (hunk 2 above)
Comment (astrozzc) [reply akamai/data/stage/base_rules.json]: Same here, but I will confirm with Nicole and Ryan.

File: akamai/data/stage/base_rules.json (hunk 1 above)
Comment (astrozzc) [reply akamai/data/stage/base_rules.json]: Based on @ryelo 's suggestion, this redirect is not needed.

File: akamai/data/stage/base_rules.json (hunk 2 above)
Comment (astrozzc) [reply akamai/data/stage/base_rules.json]: Updated the link with *, making the redirect rule more generic seems better @ryelo

File: akamai/data/stage/base_rules.json (hunk 3 above)
Comment (astrozzc) [reply akamai/data/stage/base_rules.json]: Thanks, I have updated it, see if @ryelo has some other opinion

File: akamai/data/stage/base_rules.json (hunk 4 above)
Comment (astrozzc) [reply akamai/data/stage/base_rules.json]: Removed :)

**Review by ryelo — APPROVED** [review]
I couldn't find any glaring issues here. LGTM.

**Review by kruai — APPROVED** [review]
LGTM

**Pull request merged**